src/smhi/metobs.py

```diff
@@ -57,50 +56,54 @@ def get_parameters(self, version: Union[str, int] = "1.0"):
             )
 
         self.version = version
-        self.parameter = MetObsParameterV1(self.available_versions)
+        self.parameters = MetobsParametersV1(self.available_versions)
 
-    def get_stations(self, parameter: str = None, parameter_title: str = None):
+    def get_stations(self, parameters: str = None, parameters_title: str = None):
```

Comment:
Sorry for being pedantic, but I think for function parameters (and arguments) it makes more sens to keep the singular naming, e.g. parameter, as only one parameter is sent in.